submissions/SamVal007/Memory_Pair_Game/main.js

```diff
@@ -0,0 +1,134 @@
+let similarCards = [];
+let blockedClick = false;
+const images = ['Dart_Weider', 'lyuk-skajuoker', 'R2-D2', 'c-3po',
+                'printsessa-leya-organa', 'enakin-skajuoker', 'obi-van-kenobi',
+                 'dart-sidius', 'kvaj-gon-dzhinn','joda'];
+
+const mainAttributes = {
+    container: 'container',
+    game: 'game',
+    card: 'card',
+    front: 'front_card',
+    frontImg: 'img-front_card',
+    back: 'back_card',
+    backImg: 'img-back_card',
+    flipped: 'flipped',
+    hidden: 'hidden'
+};
```

Comment:
Like for storing css classes in constants :+1:

Reply:
Thank you! :)